This PR proposes changes that would break the pipeline definition:
```
hvpa-controller-gcr-to-ar: Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/concourse/replicator.py", line 141, in render
    definition_descriptor = self._render(definition_descriptor)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/concourse/replicator.py", line 185, in _render
    'definition': factory.create_pipeline_definition(),
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/concourse/factory.py", line 88, in create_pipeline_definition
    self._apply_traits(variant)
  File "/usr/lib/python3.11/site-packages/concourse/factory.py", line 172, in _apply_traits
    for step in transformer.inject_steps():
  File "/usr/lib/python3.11/site-packages/concourse/model/traits/publish.py", line 475, in inject_steps
    for img in self.trait.dockerimages():
  File "/usr/lib/python3.11/site-packages/concourse/model/traits/publish.py", line 369, in dockerimages
    yield from (
  File "/usr/lib/python3.11/site-packages/concourse/model/traits/publish.py", line 377, in <genexpr>
    if matches_platform(platform=platform, image_args=args)
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/concourse/model/traits/publish.py", line 356, in matches_platform
    if (platforms := image_args.get('platforms', None)) is None:
                     ^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'

```

--- a/.ci/pipeline_definitions
+++ b/.ci/pipeline_definitions
@@ -5,7 +5,8 @@ hvpa-controller:
         preprocess:
           'inject-commit-hash'
         inject_effective_version: true
-      component_descriptor: ~
+      component_descriptor:
+        ocm_repository: europe-docker.pkg.dev/gardener-project/snapshots
       publish:
         oci-builder: docker-buildx
         platforms:
@@ -13,7 +14,7 @@ hvpa-controller:
         - linux/arm64
         dockerimages:
           hvpa-controller:
-            image: 'eu.gcr.io/gardener-project/gardener/hvpa-controller'
+            image: europe-docker.pkg.dev/gardener-project/snapshots/gardener/hvpa-controller
             dockerfile: 'Dockerfile'
             inputs:
               repos:
@@ -27,7 +28,9 @@ hvpa-controller:
   jobs:
     head-update:
       traits:
-        component_descriptor: ~
+        component_descriptor:
+          ocm_repository_mappings:
+            - repository: europe-docker.pkg.dev/gardener-project/releases
         draft_release: ~
     pull-request:
       traits:
@@ -36,6 +39,12 @@ hvpa-controller:
       traits:
         version:
           preprocess: 'finalize'
+        component_descriptor:
+          ocm_repository: europe-docker.pkg.dev/gardener-project/releases
+        publish:
+          dockerimages:
+            hvpa-controller:
+              image: europe-docker.pkg.dev/gardener-project/releases/gardener/hvpa-controller
         release:
           nextversion: 'bump_minor'
           git_tags:

--- a/Makefile
+++ b/Makefile
@@ -14,10 +14,10 @@
 
 VERSION             := $(shell cat VERSION)
 PACKAGES            :="$(go list -e ./... | grep -vE '/tmp/|/vendor/')"
-REGISTRY            := eu.gcr.io/gardener-project/gardener
+REGISTRY            := europe-docker.pkg.dev/gardener-project/public
 REPO_ROOT           := $(shell dirname "$(realpath $(lastword $(MAKEFILE_LIST)))")
 
-IMAGE_REPOSITORY    := $(REGISTRY)/hvpa-controller
+IMAGE_REPOSITORY    := $(REGISTRY)/gardener/hvpa-controller
 IMAGE_TAG           := $(VERSION)
 
 # Image URL to use all building/pushing image targets

